packages/nestjs/src/lib/create-automapper-providers.util.ts

```diff
@@ -8,40 +8,13 @@ export function createAutomapperProviders(
   forRootOptions: AutomapperModuleOptions,
   logger: Logger
 ): Provider[] {
-  const {
-    options,
-    globalErrorHandler,
-    globalNamingConventions,
-    singular = false,
-  } = forRootOptions;
+  const { options, globalErrorHandler, globalNamingConventions } =
+    forRootOptions;
```

Comment:
For better compatibility, I think we should warn the users (`console.warn`) about the deprecation if they explicitly use the `singular` flag.